src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -558,15 +569,40 @@ private[fastq] object FastqDemultiplexer {
   /** Stores the minimal information for a single template read. */
   case class DemuxRecord(name: String, bases: String, quals: String, molecularBarcode: Option[String],
                          sampleBarcode: Option[String], readNumber: Int, pairedEnd: Boolean, comment: Option[String],
-                         originalBases: Option[String] = None, originalQuals: Option[String] = None)
+                         originalBases: Option[String] = None, originalQuals: Option[String] = None) {
+
+    /** A function to mask bases that have a quality score less than or equal to a specified threshold
```

Comment:
No need to say "A function"
```suggestion
    /** Masks bases that have a quality score less than or equal to a specified threshold.
```